test/runtests.jl

```diff
@@ -0,0 +1,109 @@
+using PyPlot, Base.Test, PyCall
+
+#Checking if there's a GUI
+@test typeof(pygui()) == Symbol
+pygui(false)
+pygui(true)
```

Comment:
It seems like it would be better to set up the tests to run in non-interactive, non-GUI mode, with the Agg backend.